flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
+}
```

Comment:
Why not build from source?

Reply:
I'd like to eventually, but for now I'm trying to just follow the path of least resistance and do what the tauri docs and other apps I've seen do